docs/topics/signals.txt

```diff
@@ -189,6 +189,20 @@ In some circumstances, the code connecting receivers to signals may run
 multiple times. This can cause your receiver function to be registered more
 than once, and thus called multiple times for a single signal event.
 
+As Django has evolved, this issue has generally less risk of presenting itself.
+Before ``AppConfig.ready`` was introduced to Django, it was possible that
+during the Django setup process during normal operation or from a management
+command, the handlers.py file would be imported multiple times, and thus,
+the signal handler would be registered each time.
+
+Another (less common but still possible) occurrence is if in your application
+code you import your handlers.py module from somewhere in your project because
```

Comment:
```suggestion
Another potential occurrence is, if your application imports
handlers.py from somewhere in your project because
```

Reply:
I've made an attempt to reduce the wording and clarity in general. Resolving for now as the words are probably out of date now, sorry!